### PR TITLE
Add support for .NET Framework packages.config

### PR DESCRIPTION
**Pull request by olsh**
Here is an example of packages.config:
https://github.com/olsh/resharper-configuration-sense/blob/master/src/Resharper.ConfigurationSense/packages.config

### CODE REVIEW

File: lib/plugins/dot-net.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  getPattern() {
+    return {
+      pathPatterns: ['/packages.config$'],
+      githubClasses: [],
+    };
+  },
+
+  parseBlob(blob) {
+    const packageRegex = new RegExp('<\\s*package\\s.*id="(.*?)".*\\/>', 'g');
```
Comment (josephfrazier) [inline lib/plugins/dot-net.js]: Could you move this RegExp into [helper-grammar-regex-collection](https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/index.js) and add a few [tests](https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/test.js) of it? (commit a3ec0f9c048873509b6c60bd05e0019d2cf32657 has an example of adding/testing a new regex)

File: test/plugins/dot-net.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    );
+  });
+
+  it('resolves Stanford.NLP.CoreNLP to https://www.nuget.org/packages/Stanford.NLP.CoreNLP', () => {
+    assert.deepEqual(
+      dotNet.resolve({ target: 'Stanford.NLP.CoreNLP' }),
+      ['https://www.nuget.org/packages/Stanford.NLP.CoreNLP'],
+    );
+  });
+});
```
Comment (josephfrazier) [inline test/plugins/dot-net.test.js]: Could you also add some fixtures here in order to make sure that the `NET_PACKAGE` regex matches the package name correctly? https://github.com/OctoLinker/browser-extension/blob/02c0d0654a1436cdc90da33c92f82eb05463e8c2/packages/helper-grammar-regex-collection/test.js#L6

**Comment by josephfrazier** [comment]
Thanks, this work looks really good! I took the liberty of making one small tweak to the README before merging, rather than making you do it. Hope you don't mind :)

**Comment by olsh** [comment]
@josephfrazier thanks! :)

**Pull request merged**